bindings.go

```diff
@@ -0,0 +1,248 @@
+package libovsdb
+
+import (
+	"fmt"
+	"reflect"
+)
+
+var (
+	intType  = reflect.TypeOf(0)
+	realType = reflect.TypeOf(0.0)
+	boolType = reflect.TypeOf(true)
+	strType  = reflect.TypeOf("")
+	nilType  = reflect.TypeOf(nil)
+)
+
+// ErrWrongType describes typing error
+type ErrWrongType struct {
+	from     string
+	expected string
+	got      interface{}
+}
+
+func (e *ErrWrongType) Error() string {
+	return fmt.Sprintf("Wrong Type (%s): expected %s but got %s (%s)",
+		e.from, e.expected, e.got, reflect.TypeOf(e.got))
+}
+
+// NewErrWrongType creates a new ErrWrongType
+func NewErrWrongType(from, expected string, got interface{}) error {
+	return &ErrWrongType{
+		from:     from,
+		expected: expected,
+		got:      got,
+	}
+}
+
+// ErrBadSchema describes an error in the provided columnSchema
+type ErrBadSchema struct {
+	column *ColumnSchema
+}
+
+func (e *ErrBadSchema) Error() string {
+	return fmt.Sprintf("Bad Schema: %v", e.column)
+}
+
+// NewErrBadSchema creates a new ErrBadSchema
+func NewErrBadSchema(column *ColumnSchema) error {
+	return &ErrBadSchema{
+		column: column,
+	}
+}
+
+// nativeTypeFromExtended returns the native type that can hold a value of an
+// Extended type
+func nativeTypeFromExtended(extended ExtendedType) (reflect.Type, error) {
+	switch extended {
+	case TypeInteger:
+		return intType, nil
+	case TypeReal:
+		return realType, nil
+	case TypeBoolean:
+		return boolType, nil
+	case TypeString:
+		return strType, nil
+	case TypeUUID:
+		return strType, nil
+	default:
+		return nilType, fmt.Errorf("%s is not an native type", extended)
+	}
+}
+
+// NativeValueOf returns the native value of the atomic element
+// Usually, this is just reflect.ValueOf(elem), with the only exception of the UUID
+func NativeValueOf(elem interface{}, elemType ExtendedType) (reflect.Value, error) {
```

Comment:
I think lower case should be used (same for the other functions in this file). It should be private to the package. It is used in later commits by native.go and orm.go but they belong to the same package. Or do you have any use cases that requires the functions to be public?

Reply:
Done, thanks